Dockerfile

```diff
@@ -20,8 +20,23 @@ RUN DEBIAN_FRONTEND=noninteractive apt-get update -yqq && \
     python3-pylibacl \
     python3-pyxattr
 
+# Build dependencies specific for rdiff-backup development and testing
+RUN DEBIAN_FRONTEND=noninteractive apt-get update -yqq && \
```

Comment:
i would prefer to extract the `DEBIAN_FRONTEND=noninteractive` to be extracted into an `ENV DEBIAN_FRONTEND=noninteractive` at the beginning of the docker file instead of repeating it multiple times (especially as i always have to think twice if its only applied to `apt-get update` here or also to `apt-get install`)